plugins/utils/commands/fix-pr.md

```
@@ -0,0 +1,529 @@
+---
+description: Iteratively fix PR/MR checks and address review comments until PR/MR is mergeable
+argument-hint: "[PR/MR number] [-n N] [--skip-reviews] [--skip-rebase] [--auto-approve=no|ai|human|all]"
+---
+
+## Name
+utils:fix-pr
+
+## Synopsis
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)